docs/configs.md

```diff
@@ -56,6 +56,7 @@ Name | Description | Default Value
 <a name="sql.format.orc.write.enabled"></a>spark.rapids.sql.format.orc.write.enabled|When set to false disables orc output acceleration|true
 <a name="sql.format.parquet.enabled"></a>spark.rapids.sql.format.parquet.enabled|When set to false disables all parquet input and output acceleration|true
 <a name="sql.format.parquet.read.enabled"></a>spark.rapids.sql.format.parquet.read.enabled|When set to false disables parquet input acceleration|true
+<a name="sql.format.parquet.smallFiles.enabled"></a>spark.rapids.sql.format.parquet.smallFiles.enabled|When set to true, handles reading multiple small files within a partition more efficiently by combining multiple files on the CPU side before sending to the GPU. Recommended unless user needs mergeSchema option or has files with mixed legacy date/timestamps (spark.sql.legacy.parquet.datetimeRebaseModeInRead)|true
```

Comment:
Can we have a follow on issue to try and more cleanly handle schema evolution and datetimeRebaseMode?

Reply:
actually I changed it so that it does handle mixed datetimeRebaseMode by splitting the files into separate batches if it finds the mode on the files are different. I'll update the description here. 
I'll file a follow on to improve schema evolution.